.github/workflows/ci_build_android.yml

```diff
@@ -2,6 +2,12 @@ name: Build Android library
 
 on:
   workflow_call:
+    inputs:
+      production:
+        description: Builds android libraries for production release (with additional optimizations)
+        default: false
```

Comment:
```suggestion
        default: "false"
```

Reply:
that should be fine because in a reusable workflow we can specify a type (l10). For

Reply:
ok, thanks!